SConstruct

```diff
@@ -606,6 +606,12 @@ o.Add(
 	"$INSTALL_PREFIX/include",
 )
 
+o.Add(
+	"INSTALL_RESOURCE_DIR",
```

Comment:
This seems very generic, but we're using it for a very specific purpose. Would it be better to have something more explicit like `INSTALL_PXRPLUGIN_DIR`?

Reply:
I renamed the variable to `INSTALL_USD_RESOURCE_DIR` as it's more descriptive while still generic enough.